site/_docs/howto.md

```diff
@@ -927,12 +927,6 @@ thread to discuss.
 Julian
 {% endhighlight %}
 
-Use the [Apache URL shortener](https://s.apache.org) to generate
-shortened URLs for the vote proposal and result emails. Examples:
-[s.apache.org/calcite-1.2-vote](https://s.apache.org/calcite-1.2-vote) and
-[s.apache.org/calcite-1.2-result](https://s.apache.org/calcite-1.2-result).
```

Comment:
It seems there are no steps need to use this, I propose to remove it to make the document clean.

Reply:
+1, Since the release script generates the emails, I think most RMs will just copy that verbatim, so the shortening of links seem redundant.

Reply:
My understanding is that these steps are for creating mnemonic URLs that people can easily find and refer to it in future. I don't think was ever meant to be used for the voting process it self but for later reference to it if needed. I don't have a strong opinion on keeping or removing them but they do serve a purpose. For instance, typing the following just works and brings me what I need:
* https://s.apache.org/calcite-1.34.0-vote
* https://s.apache.org/calcite-1.34.0-result

Reply:
I also agree that it served for some purpose. The problem now is that it somethings confuses new RMs (this has happened during 1.35.0 releasing: https://lists.apache.org/thread/nv2xclvgrw84866o0q2ngdx4c2dg6pqq).

Reply:
If the explanation that I gave above clarifies a bit the situation then we could enrich the doc accordingly. If you feel that there might be still confusion then feel free to remove the section altogether.